client/src/components/Dashboard/Dashboard.jsx

```diff
@@ -82,20 +93,38 @@ function Dashboard() {
           {/* Background Image */}
         </div>
         {spotifyProfile && 
-          <ProfileCard spotifyProfile={spotifyProfile} spotifyToken={spotifyToken}/>
+          <ProfileCard 
+            spotifyProfile={spotifyProfile} 
+            profile={profile}
+          />
         }
         <div className="dashboard-right">
           <div className="nav">
             <div className="links">
               <button onClick={() => {
                 setPlaylistShow(true)
                 setFavoriteShow(false)
+                setFollowersShow(false)
+                setFollowingShow(false)
               }} className={`${playlistShow ? 'tab-show' : ''}`}>Playlists</button>
               <button onClick={() =>{
                 setPlaylistShow(false)
                 setFavoriteShow(true)
+                setFollowersShow(false)
+                setFollowingShow(false)
               }} className={`${favoriteShow ? 'tab-show' : ''}`}>Favorites</button>
-              <button>Followers</button>
+              <button onClick={() => {
```

Comment:
These buttons seem to be getting cumersome. Is it possible to create subclasses of buttons that have the defaults you need? 🤔